src/noshow/__init__.py

```diff
@@ -0,0 +1,6 @@
+import tomli
+
+# load mute_period from toml file
+with open("pyproject.toml", mode="rb") as fp:
```

Comment:
Better to refer to the toml file relative to this file using `Path(__file__).parent ...`